handbook/company/pricing-features-table.yml

```diff
@@ -636,10 +637,13 @@
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/17315
     - description: Automatically re-deploy configuration profiles when they're not installed.
     - description: Deploy configuration profiles on iOS/iPadOS. Coming soon (2024-07-15).
+    - description: See a list of the upcoming MDM commands and scripts in unified queue. Coming soon (2024-07-15)
+      moreInfoUrl: https://github.com/fleetdm/fleet/issues/15920
+    - description: MDM commands for iOS/iPadOS are coming soon (2024-07-15).
   productCategories: [Device management]
   pricingTableCategories: [Device management]
-- industryName: Self-remediation
-  description: Provide resolution instructions for end users through Fleet Desktop that suggest how an end user can fix a posture issue themselves.
+- industryName: Device remediation
```

Comment:
Renamed to **"Device remediation"** and updated the description/tooltip to include information about self and automated remediation.